src/routes/assignBusToRoute.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+import Buses from '../controllers/assignBusToRoute'
+
+const router = express.Router()
+
+router.put('/api/routes/:routeId/:busId', Buses.assignBus)
+router.get('/api/routes/buses/:routeId', Buses.busAssociatedToRoute)
+
```

Comment:
you can change assign bus endpiont as suggested 
ex: /api/routes/:routeId/bus/:busId

Reply:
please review the changes you requested